Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -31,7 +31,7 @@
     export class OverviewController implements IOverviewController {
         private storageKey = "it-contract-overview-options";
         private orgUnitStorageKey = "it-contract-overview-orgunit";
-        private gridState = this.gridStateService.getService(this.storageKey, this.user.id);
+        private gridState = this.gridStateService.getService(this.storageKey, this.user.id, this.user.currentOrganizationId);
```

Comment:
Kan vi ikke ændre til at det er this.user der bliver givet med her? Så har vi bare et argument som vi så adskiller inde i gridStateService